src/gluonts/dataset/jsonl.py

```diff
@@ -76,3 +79,31 @@ def __len__(self):
                 chunk.count(b"\n") for chunk in iter(read_chunk, b"")
             )
             return file_len
+
+
+@dataclass
+class JsonLinesWriter(DatasetWriter):
+    use_gzip: bool = True
+    suffix: str = ".json"
+
+    def write_to_file(self, dataset: Dataset, path: Path) -> None:
+        from .common import serialize_data_entry
```

Comment:
Why the local import, and not top-level?

Reply:
Circular imports!